### ISSUE TITLE
Support for feeds that are a static list of accounts

### ISSUE THREAD

**Issue by alimony** [bluesky-social/feed-generator#26]
I’m curious if there is any way with the current support to create a custom feed that is just a list of accounts. (Consuming the feed would populate it with posts from those accounts.) This would allow for easy implementation of communities/lists functionality without having to run a server since such a feed could be hosted statically. It seems to me such ”algorithms” are going to be popular but take a lot of overhead and maintenance.

**Comment by dholms** [comment]
Yeah totally possible - altho something so simple may just be implemented as a "list" (similar to twitter lists)
